app/src/main/webapp/js/services.js

```diff
@@ -42,51 +42,84 @@ esbMessageAdminServices.service('EsbMessageService', ['$http', '$q', 'Globals',
     };
 
     self.getSyncKeysTree = function() {
-        return $http({method: 'GET', url: "api/key/tree/Entities"});
+        return self.getTree("Entities");
     };
 
     self.getSearchKeysTree = function() {
-        return $http({method: 'GET', url: "api/key/tree/SearchKeys"});
+        return self.getTree("SearchKeys");
     };
 
-    self.addKey = function(argParentId, argName, argType, argValue) {
+    self.respondSuccess = function(argResponse) {
+        if(argResponse.status==="Success") {
+            return argResponse.tree;
+        } else {
+            messageCenterService.add('danger', argResponse.errorMessage, {status: messageCenterService.status.permanent});
+            return null;
+        }
+    }
+
+    self.respondError= function()  {
+        messageCenterService.add('danger', 'Unable to contact server!', {status: messageCenterService.status.permanent});
+        return null;
+    }
+
+    self.getTree = function(argType) {
+        return $http.get("api/key/tree/"+argType).
+        success(function(response){
+            self.respondSuccess(response);
+        }).
+        error(function(){
+            self.respondError();
+        });
```

Comment:
I'm not sure about how strict you guys have been about style standards, but to me this is confusing to read since `return` and `success(` and `error(` are all at the same indentation.